src/legacy/core_plugins/telemetry/public/components/opt_in_banner_component.tsx

```diff
@@ -45,12 +45,18 @@ export class OptInBanner extends React.PureComponent<Props> {
         <EuiFlexGroup gutterSize="s" alignItems="center">
           <EuiFlexItem grow={false}>
             <EuiButton size="s" onClick={() => this.props.optInClick(true)}>
-              <FormattedMessage id="telemetry.welcomeBanner.yesButtonLabel" defaultMessage="Yes" />
+              <FormattedMessage
+                id="telemetry.welcomeBanner.yesButtonLabel"
```

Comment:
I'm not sure when changing `telemetry.welcomeBanner.yesButtonLabel` if we also have to remove from `ja-JP.json` and `zh-CN.json` files.

Reply:
working w/ @Bamieh on that right now ...